src/main/java/org/example/Person.java

```diff
@@ -0,0 +1,48 @@
+package org.example;
+
+import java.util.List;
+
+public class Person {
+    List<Hobby> hobbies;
+    private final String name;
+    private final Integer age;
+
+    public Person(String name, Integer age, List<Hobby> hobbies)
+    {
+        this.name = name;
+        this.age = age;
+        this.hobbies = hobbies;
+    }
+
+    @override
```

Comment:
this code is definitely not compiling...
```suggestion
    @Override
```